### DISCUSSION TITLE
Need typing indicator for when someone is using chat

### DISCUSSION THREAD

**Discussion by metalgearsloth** [space-wizards/space-station-14#4515] [Feature Request]
Also bikeshed on discord if you disagree about it existing.

**Comment by PaulRitter** [comment]
So i'm going to move this to a discussion so we can better bikeshed about this.

I think a typing indicator is neat, but i could see the argument of it being abused by powergamers. I propose a different idea im in no way invested in and just came to mind:

A "typing button" that shows a temporary popup over your head indicating that you want to tell someone something. Might as well be a speechbubble with an exclamationmark like the og typing indicators.

**Reply by metalgearsloth** [reply]
Yeah that was my concern: antag types to you, waits for typing then attacks which I can't think of an easy solution to solve, but typing indicator is nice as you can tell whether the person is afk or a slow typer.

**Reply by PaulRitter** [reply]
As i said, you would be able to combat half of the issue by just showing the popup on button press, since this way you cannot be sure that they are indeed typing at that moment.

**Comment by VictorVow** [comment]
Good feature. Arguments about powergamers abusing it can be ignored in my opinion because it is very helpful in 99% of conversations and for streamlining RP.
